spec/factories/sponsors.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
```

Comment:
This is probably a generated file. You might want to populate it with some Faker data or just remove it for now.

Reply:
I modified this to just assign a dummy sponsor name.